server/plugin.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"sync"
+
+	"github.com/mattermost/mattermost-server/plugin"
+)
+
+const (
+	WebexOAuthSessionKey = "webex_session_"
```

Comment:
nit: In Mattermost code we normally use snake case all caps for constants (As a plugin not need to follow exactly the same code styling, so feel free to ignore this comment unless other core developer has a stronger opinion).

Reply:
Good to know.  Personally I prefer all cap snake case for constants but I got tired of golint saying not to use ALL_CAPS in Go names.  Which eventually broke my habit of using them. :(

Reply:
FWIW, I think we should /fix/ this in Mattermost code, even at the expense of breaking consistency. It's painful for all the reasons @stevepartridge ran into, and very "un-golike". I support doing it right here, especially given that this is a completely separate repository.